dbio/dss/upload_to_cloud.py

```diff
@@ -103,10 +103,10 @@ def upload_to_cloud(file_handles, staging_bucket, replica, from_cloud=False, log
                 )
                 sums = fh.get_checksums()
                 metadata = {
-                    "hca-dss-s3_etag": sums["s3_etag"],
-                    "hca-dss-sha1": sums["sha1"],
-                    "hca-dss-sha256": sums["sha256"],
-                    "hca-dss-crc32c": sums["crc32c"],
+                    "dbio-dss-s3_etag": sums["s3_etag"],
```

Comment:
this might cause breaking changes to upload to the DSS due to the DSS looking for `hca-dss-*` perhaps we should rename this and remove the mention of platform -> `dss-*`

Reply:
Done in #17